src/Quartz/Impl/AdoJobStore/MySQLDelegate.cs

```diff
@@ -34,5 +34,10 @@ protected override string GetSelectNextTriggerToAcquireSql(int maxCount)
         {
             return SqlSelectNextTriggerToAcquire + " LIMIT " + maxCount;
         }
+
+        protected override string GetSelectNextMisfiredTriggersInStateToAcquireSql(int maxCount)
+        {
+            return SqlSelectHasMisfiredTriggersInState + " LIMIT " + maxCount;
```

Comment:
Sori lahma,

I miss this case,  i found this line
int maxMisfiresToHandleAtATime = (recovering) ? -1 : MaxMisfiresToHandleAtATime;
20 or -1

I just commited.

Tks.